react/components/SliderContext.tsx

```diff
@@ -1,30 +1,44 @@
-import React, { createContext, useReducer, useContext, ReactNode } from 'react'
+import React, { createContext, useReducer, useContext, FC } from 'react'
+import { useDevice } from 'vtex.device-detector'
 
 interface LoadAction {
-  type: 'load'
-  payload: { slidesToShow: number; deviceType: string }
+  type: 'LOAD'
+  payload: {
+    slidesToShow: number
+    deviceType: 'desktop' | 'tablet' | 'phone'
+    navigationStep: number
+  }
 }
 
 interface LoadCorrectAction {
-  type: 'loadAndCorrect'
+  type: 'LOADANDCORRECT'
```

Comment:
Usually action types are snake_case
```suggestion
  type: 'LOAD_AND_CORRECT'
```